qlib/contrib/data/highfreq_handler.py

```diff
@@ -124,14 +124,37 @@ def __init__(
         day_length=240,
     ):
         self.day_length = day_length
+
+        def check_transform_proc(proc_l):
```

Comment:
This function duplicates with the previous one in this file. Pleases shared them